components/brave_wallet/common/string_utils_unittest.cc

```diff
@@ -60,47 +60,50 @@ TEST(StringUtilsUnitTest, Base10ValueToUint256) {
 TEST(StringUtilsUnitTest, Base10ValueToInt256) {
   int256_t out;
   ASSERT_TRUE(Base10ValueToInt256("0", &out));
-  ASSERT_EQ(out, (int256_t)0);
+  EXPECT_EQ(out, int256_t(0));
   ASSERT_TRUE(Base10ValueToInt256("1", &out));
-  ASSERT_EQ(out, (int256_t)1);
+  EXPECT_EQ(out, int256_t(1));
   ASSERT_TRUE(Base10ValueToInt256("-1", &out));
-  ASSERT_EQ(out, (int256_t)-1);
+  EXPECT_EQ(out, int256_t(-1));
   ASSERT_TRUE(Base10ValueToInt256("12345678910", &out));
-  ASSERT_EQ(out, (int256_t)12345678910);
+  EXPECT_EQ(out, int256_t(12345678910));
   ASSERT_TRUE(Base10ValueToInt256("-12345678910", &out));
-  ASSERT_EQ(out, (int256_t)-12345678910);
+  EXPECT_EQ(out, int256_t(-12345678910));
 
+  // int types in Boost uses extra sign bit so int256_t precision bit is 256
+  // bits.
   // Max int256 value can be represented
   int256_t expected_val = std::numeric_limits<int256_t>::max();
   ASSERT_TRUE(
-      Base10ValueToInt256("5789604461865809771178549250434395392663499233282028"
-                          "2019728792003956564819967",
+      Base10ValueToInt256("115792089237316195423570985008687907853269984665640"
```

Comment:
same as above, I think for all of these methods we want to limit it to 255 bit with sign, not 256 bit with sign

Reply:
fixed in https://github.com/brave/brave-core/commit/54300c8be0ff9fe54ce35776634408d72965c54b

Reply:
seems like it's still different, maybe something got lost in a rebase? expected val should use the const and the input value should not change

Reply:
yea, looks like those changes were lost, but that commit still has different input values, shouldn't they be the same as the original test?